src/handle_wrap.cc

```diff
@@ -18,31 +18,47 @@ using v8::Value;
 
 
 void HandleWrap::Ref(const FunctionCallbackInfo<Value>& args) {
-  HandleWrap* wrap = Unwrap<HandleWrap>(args.Holder());
+  HandleWrap* wrap;
+  ASSIGN_OR_RETURN_UNWRAP(HandleWrap,
+                          wrap,
+                          args.Holder(),
+                          args.GetReturnValue().Set(false));
```

Comment:
The check is for when `wrap == nullptr`, so `wrap->IsAlive()` aborts in that case.